pyproject.toml

```diff
@@ -43,6 +54,7 @@ all = [
     "pyvista>=0.40.1",
     "torch-harmonics>=0.6.2",
     "onnx>=1.14.0",
+    "nvidia-modulus[launch]"
```

Comment:
What are the deps above this? Those are docker deps? Also, why no [dev] in [all]?

Reply:
Thanks! added [dev] to [all]. Also, yes the other dependencies are to get the full functionality of the Core (basically docker deps). We added the instructions for them here: https://docs.nvidia.com/deeplearning/modulus/getting-started/index.html (under the pip section)